research_town/dbs/db_base.py

```diff
@@ -38,7 +38,8 @@ def add(self, data: T) -> None:
             self.data_class.__name__, [data.model_dump(exclude_none=True)], embeddings
         )
         logger.info(
-            f"Creating instance of '{data.__class__.__name__}': '{data.model_dump()}'"
+            # f"Creating instance of '{data.__class__.__name__}': '{data.model_dump()}'"
```

Comment:
should we change this? What is the motivation for not shwoing model_dump? Toolong?